lib/blockscore/errors/api_error.rb

```diff
@@ -15,19 +17,29 @@ class APIError < Error
     # APIError - Indicates an error on the server side (HTTP 5xx)
     # AuthenticationError - Indicates an authentication error (HTTP 401)
     def initialize(response)
-      body = JSON.parse(response.body, symbolize_names: true)
+      @http_body = JSON.parse(response.body, symbolize_names: true)
 
-      @message = body[:error][:message]
+      @message = error_advice[:message]
```

Comment:
Yes  - I ran mutant - and mutant suggests it, but the tests are untrustworthy. So I asked the question re is it always present in Hipchat and did not get an answer yet from @backus.